docs/src/changelog.md

```diff
@@ -12,7 +12,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Deployment notes
 
-If your currently installed *Quality-time* version is v4.10.0 or older, please read the v5.0.0 deployment notes first.
+If your currently installed *Quality-time* version is not v5.13.0, please read the [versioning policy](versioning.md) first.
```

Comment:
This is already in preparation of a major version bump?

Reply:
No, the idea is that the version overview in the versioning policy shows when versions can be skipped. I'll change the sentence to 

```suggestion
If your currently installed *Quality-time* version is not v5.13.0, please first check the upgrade path in the [versioning policy](versioning.md).
```